src/site/content/en/blog/maskable-icon/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Better PWA icons with maskable icons
+subhead: A new icon format to use adaptive icons on supporting platforms.
+description: |
+  A description of the article that will appear in search results.
+authors:
+  - tigeroakes
+date: 2019-12-13
+hero: hero.png
+alt: Icons contained inside white circles compared to icons covering its entire circle
+tags:
+  - post
+  - capabilities
+  - progressive-web-apps
+---
+
+## What are maskable icons? {: #what }
+
+If you've installed a Progressive Web App on a recent Android phone, you might notice the icon shows up with a white background. Android Oreo introduced adaptive icons, which display app icons in a variety of shapes across different device models. Icons that don't follow this new format are given white backgrounds.
+
+<figure class="w-figure">
+  <img src="homescreen-any.png" alt="" style="max-width: 400px;">
```

Comment:
Need an alt description for this and all images. Using `alt=""` hides the image from screen readers, but they would still hear the figcaption, which could be confusing.

Although you're providing a caption, unfortunately we can't use that as a replacement for alt text.
https://www.scottohara.me/blog/2019/01/21/how-do-you-figure.html#a-figcaption-is-not-a-substitute-for-alt-text

I usually suggest to think of alt text as if you were describing the image to someone over the phone. What are the key visual bits you'd want to convey. The caption, on the other hand, provides additional context that helps explain how the image relates to the rest of the document.